xml/System.Data.Common/DbException.xml

```diff
@@ -321,7 +321,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Indicates whether the error represented by this <see cref="T:System.Data.Common.DbException" /> could be a transient error, i.e. if retrying the triggering operation may succeed without any other change. Examples of transient errors include failure to acquire a database lock, networking issues. This allows automatic retry execution strategies to be developed without knowledge of specific database error codes.</summary>
         <value>To be added.</value>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
        <summary>Indicates whether the error represented by this <see cref="T:System.Data.Common.DbException" /> could be a transient error, i.e. if retrying the triggering operation may succeed without any other change.</summary>
        <value>
          <see langword="true" /> if the error represented could be a transient error; <see langword="false" /> otherwise.</value>
        <remarks>
          <format type="text/markdown">>![CDATA[

## Remarks

Examples of transient errors include failure to acquire a database lock, networking issues. This allows automatic retry execution strategies to be developed without knowledge of specific database error codes.

          ]]></format>
        </remarks>
```